src/github-repo-event-notifier.coffee

```diff
@@ -32,12 +32,16 @@
 #   patcon
 #   parkr
 
+_ = require('lodash')
 inspect = (require('util')).inspect
 url = require('url')
 querystring = require('querystring')
-eventActions = require('./event-actions/all')
 eventTypesRaw = process.env['HUBOT_GITHUB_EVENT_NOTIFIER_TYPES']
 eventTypes = []
+customActions = {}
+try
+  customActions = require.main.require('../../../scripts/github/custom-actions')
```

Comment:
Can we always assume that this project will be installed into `my-hubot\node_modules\hubot-github-repo-event-notifier\src`? It seems like a very fragile assumption to me as node modules are nested if they are a dependency of a dependency.